tests/test_export.py

```diff
@@ -172,6 +172,57 @@ def spec_shapefactor():
     return spec
 
 
+def spec_integer_data():
```

Comment:
@kratsg There is probably a more concise / intelligent way to write this spec + test, but I just grabbed the example failing spec that @lukasheinrich provided and went with it tonight. So revision requests welcome!

Reply:
An alternative way to do this is to put the function in `conftest.py` and decorate it with `@pytest.fixture`, and then you can make it available in `test_integer_data` by adding it to the signature:
```
def test_integer_data(mocker, spec_integer_data):
    channel_spec = spec_integer_data["channels"][0]
```

Reply:
Easiest way is to use the fixture I created for local test data. Move the spec into `tests/test_export/workspace_integer_data.json` and then change the test (dropping the `spec_integer_data` completely)

```
def test_integer_data(datadir, mocker):
    """
    Test that a spec with only integer data will be written correctly
    """
    spec = json.load(open(datadir.join('workspace_integer_data.json')))
    channel_spec = spec["channels"][0]
    mocker.patch('pyhf.writexml._ROOT_DATA_FILE')

    channel = pyhf.writexml.build_channel(spec, channel_spec, {})
    assert channel
```

Reply:
@kratsg This is done and ready for review then.